packages/office-ui-fabric-react/src/components/GroupedList/GroupFooter.styles.ts

```diff
@@ -0,0 +1,28 @@
+import { IGroupFooterStyleProps, IGroupFooterStyles } from './GroupFooter.types';
+import { getGlobalClassNames, FontSizes } from '../../Styling';
+
+const GlobalClassNames = {
+  root: 'ms-groupFooter',
+  link: 'ms-Link'
+};
+
+export const getStyles = (props: IGroupFooterStyleProps): IGroupFooterStyles => {
+  const { theme, className } = props;
+  const classNames = getGlobalClassNames(GlobalClassNames, theme!);
+
+  return {
+    root: [
+      classNames.root,
+      {
+        position: 'relative',
+        padding: '5px 38px',
+        selectors: {
+          [`:global(.${classNames.link}`]: {
```

Comment:
This one is nasty. I know you're just repeating the logic, so it's ok for the moment, but I think there's a bug here.

`getGlobalClassNames` will look at theme to identify if the theme doesn't want global class names. If it doesn't, then the value will be undefined, making the rule render as `.undefined { ... }` or maybe just a blank period as a selector.

A better approach? We invest in a simple "generate me a unique classname" api in `merge-styles`. Link exports const LinkPartClassNames, which we reference here, rather than the `ms-Link` name. This enforces that Links from this instance of Fabric components, and not some other version of Link from another copy of Fabric on the page, or from some other component that likes `ms-Link` classname, will be affected.